seatunnel-connectors-v2/connector-jdbc/pom.xml

```
@@ -30,12 +30,40 @@
     <artifactId>connector-jdbc</artifactId>
 
     <properties>
-        <phoenix.version>5.2.5-HBase-2.x</phoenix.version>
         <mysql.version>8.0.16</mysql.version>
         <postgresql.version>42.3.3</postgresql.version>
         <dm-jdbc.version>8.1.2.141</dm-jdbc.version>
+        <phoenix.version>5.2.5-HBase-2.x</phoenix.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.aliyun.phoenix</groupId>
+                <artifactId>ali-phoenix-shaded-thin-client</artifactId>
+                <version>${phoenix.version}</version>
+                <scope>provided</scope>
+            </dependency>
+            <dependency>
+                <groupId>mysql</groupId>
+                <artifactId>mysql-connector-java</artifactId>
+                <version>${mysql.version}</version>
+                <scope>provided</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.postgresql</groupId>
+                <artifactId>postgresql</artifactId>
+                <version>${postgresql.version}</version>
+                <scope>provided</scope>
+            </dependency>
+            <dependency>
+                <groupId>com.dameng</groupId>
+                <artifactId>DmJdbcDriver18</artifactId>
+                <version>${dm-jdbc.version}</version>
+            </dependency>
```

```suggestion
            <dependency>
                <groupId>com.dameng</groupId>
                <artifactId>DmJdbcDriver18</artifactId>
                <version>${dm-jdbc.version}</version>
                <scope>provided</scope>
            </dependency>
```
Then delete version from <version># line83